nautobot/docs/additional-features/caching.md

```diff
@@ -156,9 +155,17 @@ For more details on how to configure Cacheops to use Redis Sentinel see the docu
 
 #### `celery` Sentinel Configuration
 
-Celery Sentinel configuration is controlled by 4 variables `BROKER_URL`, `BROKER_TRANSPORT_OPTIONS`, `RESULT_BACKEND`, and `RESULT_BACKEND_TRANSPORT_OPTIONS`.  These parameters can
-be specified in the django settings in `nautobot_config.py` by prefixing these variable names with `CELERY_`.  By default Nautobot configures the celery broker and results backend with the
-same configuration.
+!!! note
+	Celery is not directly related caching but it does utilize Redis, therefore in more advanced deployments it if Redis Sentinel is required for caching, Celery must also be configured to use Redis Sentinel to high availability.
+
+Celery Sentinel configuration is controlled by four settings within your `nautobot_config.py`:
+
+-  `CELERY_BROKER_URL`
+-  `CELERY_BROKER_TRANSPORT_OPTIONS`
+-  `CELERY_RESULT_BACKEND`
+-  `CELERY_RESULT_BACKEND_TRANSPORT_OPTIONS`
```

Comment:
Might be good to hyperlink these to the relevant documentation for each setting?

Reply:
I thought of that, but got stuck on whether to link to ours or Celery's documentation for it, seeing as the "transport" ones aren't mirrored in ours, so just decided to skip it.